src/latbin/lattice-copy.cc

```diff
@@ -174,6 +176,10 @@ int main(int argc, char *argv[]) {
     std::string lats_rspecifier = po.GetArg(1),
         lats_wspecifier = po.GetArg(2);
 
+    RspecifierOptions opts;
+    ClassifyRspecifier(lats_rspecifier, NULL, &opts);
+    bool sorted = opts.sorted || opts.called_sorted;
```

Comment:
just the 'sorted' option is relevant here. (called_sorted is an assertion that the program will access them in sorted order, which is only relevant for random-access readers)>

Reply:
Makes sense, thanks! Fixed.